src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -40,7 +30,7 @@ Rules generally consist of three parts:
 ## Methods
 
 - _Remove_: Remove the entire field. We may choose to either set it to `null`, remove it entirely or replace it with an empty string depending on technical constraints.
-- _Mask_: Replace all characters with `*`. For creditcards this replaces everything but the last 4 digits.
+- _Mask_: Replace all characters with `*`.
 - _Hash_: Replace the matched substring with a hashed value.
 - _Replace_: Replace the matched substring with a constant placeholder value such as `[Filtered]` or `[creditcard]`. Right now this value cannot be configured.
```

Comment:
Suggest replacing "Right now" with "Currently" (it's what we're using in other places, so easier to find if we're searching and making changes. But it's not critical.

Reply:
the entire sentence was incorrect, removed